plugins/loader.c

```diff
@@ -265,6 +287,15 @@ static int plugin_load(struct qemu_plugin_desc *desc, const qemu_info_t *info, E
     return 1;
 }
 
+void plugin_add_qpp_cb(struct qemu_plugin_ctx *ctx, const char *name) {
+    struct qemu_plugin_qpp_cb *new_cb;
+    new_cb = qemu_memalign(qemu_dcache_linesize, sizeof(*new_cb));
+    memset(new_cb, 0, sizeof(*new_cb));
+    new_cb->name = name;
+    new_cb->plugin = ctx->name;
```

Comment:
Is the plugin name already stored in the qemu_plugin_ctx? Can we just drop this entirely?

Reply:
yeah I think this can be safely dropped